src/parse-util.c

```diff
@@ -146,3 +147,25 @@ char *format_bytes(char *buf, size_t l, uint64_t t) {
         return buf;
 
 }
+
+char *format_nsec_to_secs(char *buf, size_t l, uint64_t time_nsec) {
+        struct timespec ts;
+
+        ts = nsec_to_timespec(time_nsec);
+        snprintf(buf, l, "%lld.%.6ld", (long long) ts.tv_sec, ts.tv_nsec / 1000L);
```

Comment:
hmm maybe steal systemd's format_timespan() for this?

Reply:
I did just that.